src/Engine/EngineBase.php

```diff
@@ -143,6 +148,46 @@ public function getValidLangs( bool $withNames = false ): array {
 		return $list;
 	}
 
+	/**
+	 * Get line detection models accepted by the engine
+	 * @param bool $onlyLineIds Whether to return only the line detection model IDs
+	 * @param bool $onlyLineIdLangs Whether to return only the line detection model IDs lang codes
+	 * @return string[] Line detection model lang codes or model IDs or model ID names
+	 */
+	public function getValidLineIds( bool $onlyLineIds = false, bool $onlyLineIdLangs = false ): array {
```

Comment:
I'm not sure Line IDs should be part of the EngineBase; they're an attribute just of Transkribus, so shouldn't be at this higher level.